File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    //Retrieves user's desired height value between 1 and 8
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);


    //Iterates through rows of bricks
    for (int i = 0; i < height; i++)
    {
        //Displays number of spaces
        for (int j = 0; j < (height - (i + 1)); j++)
        {
            printf(" ");
        }

        //Displays number of hashes
        for (int k = 0; k < (i + 1); k++)
        {
            printf("#");
        }

        printf("  ");

        for (int m = 0; m < (i + 1); m++)
        {
            printf("#");
        }

        //Displays space
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

# Initlizes height to 0 to enter while loop
height = 0

# Keep prompting user until height between 1-8 is entered
while height < 1 or height > 8:
    height = get_int("Height: ")

# Iterate through each layer and add hashes, space, and formatting accordingly 
for i in range(height):
    for j in range(height - (i + 1)):
        print(" ", end='')
    for k in range(i + 1):
        print("#", end='')
    
    print("  ", end='')
    
    for m in range(i + 1):
        print("#", end='')
    
    print("\n", end='')